File: Modul 6/Soal 1/PRAK501-2210817220007-AISYA ANINDIA SEPTHA.py
def MaxBilangan(e, f, g, h) :
    e = int(e); f = int(f); g = int(g); h = int(h)
    if (e > f and e > h and e > h) :
        return e
    elif (f > e and f > g and f > h) :
        return f
    elif (g > e and g > f and g > h) :
        return g
    elif (h > e and h > f and h > g) :
        return h
a, b, c, d = input().split()
a = int (a)
b = int (b)
c = int (c)
d = int (d)
hasil = MaxBilangan(a, b, c, d)
print(hasil)

File: Modul 6/Soal 1/PRAK501-2210817220007-Aisya Anindia Septha.c
#include <stdio.h>
int MaxBilangan (int e, int f, int g, int h){
    if (e > f && e > g && e > h)
        return e;
    else if (f > e && f > g && f > h)
        return f;
    else if (f > e && g > f && g > h)
        return g;
    else if (h > e && h > f && h > g)
        return h;
}
int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan (a, b, c, d);
    printf("%d", hasil);
    return 0;
}
